tests/resources/adhoc_routing/output_events.xml

```diff
@@ -9,7 +9,7 @@
     <event time="32460" type="entered link" link="6" vehicle="0" />
     <event time="32510" type="left link" link="6" vehicle="0" />
     <event time="32510" type="PersonLeavesVehicle" person="0" vehicle="0" />
-    <event time="32510" type="arrival" person="0" link="6" legMode="car" />
+    <event time="32510" type="arrival" person="0" link="4" legMode="car" />
```

Comment:
The new value is correct (according to the plans file). Maybe that was a bug in the parallel simulation.